src/main/java/com/rednavis/vaadin/showcase/views/Constants.java

```diff
@@ -0,0 +1,13 @@
+package com.rednavis.vaadin.showcase.views;
+
+/**
+ * @author Aliaksei Labotski.
+ * @since 13/01/20.
+ */
+public class Constants {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Constants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4807567)